Change padding from 6/7px to 5px

Skaronator opened the pull request:
Its reduce the size of white space between the menu buttons. You'll see more buttons and have less scroll while looking still good.

Here a example:
![image](https://cloud.githubusercontent.com/assets/2886913/26026494/73fcf9e6-37fc-11e7-94ac-7198f2c7e4d6.png)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/openmediavault/openmediavault?pullRequest=18) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/openmediavault/openmediavault?pullRequest=18) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/openmediavault/openmediavault?pullRequest=18) it.</sub>

votdev commented:
I suggest to use

`.x-grid-cell-inner {
    padding-top: 5px;
    padding-bottom: 5px;
}`

because 4px are too small in correlation to the top and bottom toolbars of a grid.

Skaronator commented:
Even 3px are fine IMO but going to change it to 5px.

One question: What is the purpose of `<div class=" x-tree-elbow-img x-tree-elbow-line" role="presentation"></div>`

`.x-tree-icon, .x-tree-elbow-img, .x-tree-checkbox` those classes have the negative margin of `x-grid-cell-inner` padding.

Skaronator commented:
Updated. A Preprocessors like SASS or LESS might be very helpful to keep consistency over various elements.

votdev commented:
Indeed it would be great to use SASS or LESS, but this is not possible (as far as i know) without buying the whole Sencha ExtJS package because this contains the necessary tool chain and files. But i am open for contributions in this behaviour.

Skaronator commented:
A switch to ReactJS might be good but you won't have the time to do this task.

votdev commented:
It is not possible nor wanted to switch the UI framework because the code base is to big now. Also ExtJS is one of the best framework, better than AngularJS for example which i am working with, too.

votdev commented:
I have to decline this PR after testing because:

- the layout of the tree cells is totally shifted and asymetric.
- the whole UI layout is not fit anymore.

Please use /var/www/openmediavault/css/theme-custom.css to customize the UI to your needs.

Skaronator commented:
Weird. Something is wrong with the negative margin. Still don't know why it is there in first place. I've removed it and now its working just fine.

Could you verify that its working now?

Simply put this in the Dev Console when you're on the OMV Interface.

```
var css = document.createElement("style");
css.type = "text/css";
css.innerHTML = " .x-grid-cell-inner, .x-grid-cell-inner-treecolumn { padding-top: 5px; padding-bottom: 5px;}";
document.head.appendChild(css);
```

I've updated this in my fork as well but the PR isn't updating: https://github.com/Skaronator/openmediavault/commit/2cbd3772da9b2adc643ff24134f170b8eedfe4b7

votdev commented:
Fixed with https://github.com/openmediavault/openmediavault/commit/5e6b69f0851e8fab6ce41a234c7a8da67d63b6a2.

Skaronator commented:
Thanks!
